.github/workflows/repo-stale.yaml

```diff
@@ -0,0 +1,30 @@
+name: Issue Stale Check
```

Comment:
```suggestion
name: Issue stale check
```

For consistency with the "Merge conflict labeler" workflow.

Reply:
Okay, I was not sure what casing to use and too inspiration from the `App Build` Workflow :sweat_smile:
Will fix :+1: